### ISSUE TITLE
can’t delete previously faved (liked) tweet(s)

### ISSUE THREAD

**Issue by jm3** [sferik/t#358]
_on version 3.1.0:_

```
% t whoami
Screen name      @jm3
```

```
% t delete favorite 713598929389137920

Are you sure you want to remove @SquareCash's status: 
"Frankly I'm mad if you don't use it https://t.co/tZbTv1zInd" from your favorites? [y/N] y

/usr/local/bin/t: No status found with that ID.
```

<img width="602" alt="square-fave-undeleteable" src="https://cloud.githubusercontent.com/assets/12213/21738817/36b7c6a2-d441-11e6-9424-aee141965c8a.png">

**Comment by keithknox** [comment]
I hate to be that guy, but I just ran a test between my account and a test account and it seems to be favorite-ing (and un-favorite-ing) just fine. Is anyone else experiencing this? Is there something else I could be trying to reproduce the issue?

**Comment by jm3** [comment]
yes, specifically: try it on a post you favorited more than 3,200 favorites
ago.

**Comment by keithknox** [comment]
got it. i'll poke around and see what's going on.

thanks!

**Comment by sferik** [comment]
Thank you for taking the time to report this issue.

Now that X has introduced [Pay-Per-Use API Pricing](https://x.com/i/trending/2019901839370531129), I've rewritten the `t` CLI in Rust (it has also been renamed to `x`).

If you believe this issue still applies to the new CLI, please reopen this issue or open a new one with updated details. Thank you for your understanding and your contributions to this project!
